docs/tutorials/index.html

```diff
@@ -0,0 +1,70 @@
+<!--modified version of: https://docs.meilisearch.com/learn/tutorials/getting_started.html#integrate-with-your-project-->
```

Comment:
should we create a subdir `slack-history` and put the html inside as well as the whole tutorial as a README.md?

Reply:
agreed on putting the index.html in a subdir. not sure i understand what you're trying to achieve with the readme change.

Reply:
ah it is more of a gitbook organization thing but it also allows us to keep things properly namespaced:
```docs/tutorials/my-super-slack-tutorial.md```
is the same as doing:
```docs/tutorials/my-super-slack-tutorial/README.md```

My suggestion is just to use `my-super-slack-tutorial` as a directory instead of the file name. This way the tutorial text and the test file can live in the same directory:
```
docs/tutorials/my-super-slack-tutorial/README.md
docs/tutorials/my-super-slack-tutorial/index.html
```